src/main/java/edu/ie3/util/quantities/QuantityUtil.java

```diff
@@ -52,6 +75,24 @@ private QuantityUtil() {
     return Math.abs(aVal - bVal) <= absQuantityTolerance;
   }
 
+  /**
+   * Compares two {@link Quantity}s, if they are considerably equal. This is foremost important for
+   * {@link tech.units.indriya.quantity.NumberQuantity}s with internal big decimal or double values.
+   * The comparison is made on the relative difference of both quantities' value with regard to a's
+   * value. Both quantities are converted into a's unit before the comparison. Internally calls
+   * {@link QuantityUtil#isRelativelyEquivalent(Quantity, Quantity, double)} with a default
+   * tolerance of {@value DEFAULT_TOLERANCE}
+   *
+   * @param a First quantity to compare
+   * @param b Second quantity to compare
+   * @param <Q> Type of Quantity
+   * @return true, if both quantities' values differ less then the given tolerance else false
+   */
+  public static <Q extends Quantity<Q>> boolean isRelativelyEquivalent(
```

Comment:
```suggestion
  public static <Q extends Quantity<Q>> boolean isEquivalentRel(
```

See discussion above.